public/catalog/v0.1.0/openapi.json

```diff
@@ -1040,6 +1040,11 @@
             "readOnly": true,
             "title": "Support URL",
             "example": "The URL for finding support for the portfolio item"
+          },
+          "workflow_ref": {
+            "type": "string",
+            "title": "Approval Workflow ID",
+            "example": "The approval status of the portfolio item"
```

Comment:
@lindgrenj6 I believe this example should reference the current workflow assigned to the portfolio_item.  
This is the text from the Portfolio:
'The current approval workflow selected for a portfolio'

Reply:
Sorry I did not pick this up on the Portfolio, but is "current" needed in that text.  Makes me think we have historical data, which we do not.